app/models/punk_api_client.rb

```diff
@@ -0,0 +1,35 @@
+class PunkAPIClient
+  PUNK_URL = "https://api.punkapi.com/v2/beers"
+
+  attr_reader :create_beers
+
+  def initialize(beers)
+    @create_beers = beers
+  end
+
+  def self.seed_recipes
+    return beer_request(PUNK_URL)
+  end
```

Comment:
this method should be the master method. Right now, each of your methods chains to the next. We want to instead move some of the method invocations into this one
```
def self.seed_recipes
     beer_response = beer_request(PUNK_URL)
    if beer_response == true
        create_beers(beer_response) 
    end 
     # you could also check if the `create_beers` method is successful, and help return an error message 
     accordingly
  end
```